.changeset/blue-oranges-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-okta-provider': patch
```

Comment:
You need to make a separate changeset for auth-backend too, saying that the okta implementation was moved to the new module